src/contract_wrappers/modules/module_factory_wrapper.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from '@polymathnetwork/abi-wrappers';
 import { schemas } from '@0x/json-schemas';
 import assert from '../../utils/assert';
+import * as semver from 'semver';
```

Comment:
This can be written as `import semver from  'semver';`

Reply:
Issue with that is
"Module ... polymath-contract-wrappers/node_modules/@types/semver/index"' has no default export"

How do I resolve that?

Reply:
`yarn add --dev @types/semver` and add `"esModuleInterop": true,` to the compilerOptions in tsconfig.json